org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XMLFormattingOptions.java

```diff
@@ -40,27 +40,39 @@
 	// Values for QUOTATIONS
 	public static final String DOUBLE_QUOTES_VALUE = "doubleQuotes";
 	public static final String SINGLE_QUOTES_VALUE = "singleQuotes";
+
 	enum Quotations {
 		doubleQuotes, singleQuotes
-	}	
+	}
+
 	private static final String PRESERVE_EMPTY_CONTENT = "preserveEmptyContent";
 
+	/**
+	 * Options for formatting empty elements.
+	 *
+	 */
+	public static enum FormatEmptyElements {
+		expand, collapse, none;
```

Comment:
needs javadoc to explain the differences

Reply:
fixed